libarchive/test/test_write_format_zip_stream.c

```diff
@@ -71,7 +71,11 @@ DEFINE_TEST(test_write_format_zip_stream)
 	unsigned long crc;
 	unsigned long compressed_size = 0;
 	int file_perm = 00644;
+#ifdef HAVE_ZLIB_H
 	int zip_version = 20;
+#else
+	int zip_version = 10;
+#endif
```

Comment:
Something I hit running the tests locally. I should probably look into how to ensure that I build with zlib, but this can otherwise alleviate pain for anyone else who runs into this.

Reply:
This is a good fix, but please put it in a separate PR.